### PR TITLE
Update block-point-to-point-file-transfers.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md [hunk 1]
```diff
@@ -46,38 +46,22 @@ To make this work, the user must be using a supported version of a 2016 Click-to
 > [!CAUTION]
 > Users that are using older versions of Skype for Business Windows apps or Mac clients will still be able to transfer files. 
   
-## Verify and start Windows PowerShell
+## Start Windows PowerShell
 
-- **Check that you are running Windows PowerShell version 3.0 or higher**
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md:51]:
```suggestion
 > [!NOTE]
```

File: Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md [hunk 2]
```diff
@@ -46,38 +46,22 @@ To make this work, the user must be using a supported version of a 2016 Click-to
...
+  > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md:52]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**